homework_8/mkdir.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+
+parser = argparse.ArgumentParser('The mkdir utility creates the directories')
+parser.add_argument('dir', help='directory')
+parser.add_argument('-p', action='store_true',
+                    help='Create intermediate directories as required.')
+args = parser.parse_args()
+
+if args.p:
+    os.makedirs(args.dir)
```

Comment:
Если папка уже есть, то выпадет ошибка, хотя с флагом `-p` её быть не должно 
![image](https://user-images.githubusercontent.com/48160953/208510791-5b9c04ce-8839-4793-92a7-6063cfbf86cf.png)
```suggestion
    os.makedirs(args.dir, exist_ok=True)
```